health/healthcheck.py

```
@@ -131,6 +131,34 @@
 
+class LDAPHealthCheck(HealthCheck):
+    def __init__(self, app, config_provider, instance_keys):
+        super(LDAPHealthCheck, self).__init__(
+            app, config_provider, instance_keys, ["redis", "storage"]
+        )
+
+    @classmethod
+    def check_names(cls):
+        return ["LDAPHealthCheck"]
+
+    def calculate_overall_health(self, service_statuses, skip=None, notes=None):
```

Ruff (0.14.7)

Unused method argument: `notes`

(ARG002)